common/astyanax/src/main/java/com/bazaarvoice/emodb/common/cassandra/CqlCluster.java

```diff
@@ -54,6 +55,13 @@ private void registerMetrics() {
 
         Metrics metrics = _cluster.getMetrics();
 
+        JmxReporter reporter =
+                JmxReporter.forRegistry(_cluster.getMetrics().getRegistry())
+                        .inDomain(_cluster.getClusterName() + "-metrics")
+                        .build();
+
+        reporter.start();
+
         _metricRegistry.register(
                 MetricRegistry.name("bv.emodb.cql", _metricName, "ConnectionPool", "connected-to-hosts"),
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "bv.emodb.cql" 14 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.qa.bazaarvoice.com/coding_rules#rule_key=squid%3AS1192)
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "ConnectionPool" 14 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.qa.bazaarvoice.com/coding_rules#rule_key=squid%3AS1192)